style.css

```diff
@@ -0,0 +1,1043 @@
+.main-article-aside-sub-2 {
+  background: linear-gradient(to bottom right, grey, purple, red);
+  padding: 1rem;
+  color: var(--color-white);
+  display: flex;
+  justify-content: flex-end;
+  flex-direction: column;
+}
+
+.main-articles-footer {
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: row;
+  min-width: 6rem;
+  margin-top: 0.5rem;
+}
+
+.latest-news {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  min-width: 100%;
+  margin-top: 6rem;
+}
+
+.grid-header {
+  display: block;
+  width: 63rem;
+}
+
+.latest-news-grid {
+  display: grid;
+  grid-template-rows: repeat(2, 1fr);
+  grid-template-columns: repeat(4, 1fr);
+  width: 63rem;
+  grid-gap: 1.5rem;
+}
+
+.latest-news-grid img {
+  width: 100%;
+  height: 9rem;
+  cursor: pointer;
+}
+
+.latest-news-grid h4 {
+  margin: 0.4rem 0;
+  cursor: pointer;
+}
+
+.latest-news-grid h4:hover {
+  text-decoration: underline;
+  color: var(--color-orange);
+}
+
+.latest-news h2 {
+  color: var(--color-orange);
+  margin-bottom: 1rem;
+}
+
+.latest-news h2:hover {
+  color: var(--color-black);
+}
+
+.mid-article-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  min-width: 100%;
+  margin-top: 2rem;
+}
+
+.mid-article-head-container {
+  display: grid;
+  grid-template: 2rem 3fr / 1fr;
+  width: 63rem;
+}
+
+.mid-article-head-title {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.mid-article-head-title h1 {
+  width: 50%;
+}
+
+.mid-article-head-title ul {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  height: 1rem;
+}
+
+.mid-article-head-title li:not(:first-child) {
+  border-left: 1px solid var(--color-orange);
+  padding-left: 1rem;
+}
+
+.mid-article-head-title a:hover {
+  color: var(--color-black);
+}
+
+.mid-article-head-content {
+  display: grid;
+  grid-template: 1fr / repeat(4, 1fr);
+  column-gap: 1rem;
+}
+
+.mid-article-head-content-1-h1 {
+  background-color: var(--color-orange);
+  color: var(--color-white);
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  padding: 1rem;
+}
+
+.mid-article-head-content-1-h1 h1 {
+  margin-left: 4rem;
+  font-size: 1.2rem;
+}
+
+.mid-article-head-content-1 img {
+  width: 4rem;
+  height: 4rem;
+  margin-top: -40px;
+  margin-left: 1rem;
+}
+
+.mid-article-head-content-1-a1 {
+  display: flex;
+  flex-direction: column;
+  background-color: rgb(255, 235, 231);
+}
+
+.mid-article-head-content-1-a1 h6 {
+  font-size: 0.7rem;
+}
+
+.mid-article-head-content-1-body {
+  display: flex;
+  flex-direction: column;
+  line-height: 30px;
+  color: grey;
+  background-color: rgb(255, 250, 248);
+}
+
+.mid-article-head-content-1-body a:last-child {
+  border-top: 1px solid rgb(255, 205, 194);
+}
+
+.latest-deals-grid-header a:hover {
+  color: var(--color-black);
+}
+
+.mid-article-head-content-1 a:not(:last-child):hover {
+  color: var(--color-black);
+}
```

Comment:
This selector and selectors on lines 94, 132, 464, 505, and 558 all have the same style definition, group selectors to minimize code.

Reply:
Implemented